src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return cost;
+  } else if (days < 7) {
+    return (cost - smallDiscount);
+  } else {
+    return (cost - bigDiscount);
+  }
```

Comment:
```suggestion
  }
  return (cost - bigDiscount);
```

there's no need in last 'else' because if both previous checks won't pass, the function will return the last one by default